pkgs/development/tools/electron/common.nix

```diff
@@ -64,6 +64,9 @@ in
     hash = info.chromium_npm_hash;
   };
 
+  # Needed for header generation in electron 35 and above
+  env.ELECTRON_OUT_DIR = if (lib.versions.major info.version >= "35") then "Release" else null;
```

Comment:
btw this breaks `electron_33`, since `env.*` can only be a string (forgot about this). It would probably be OK if we just used `optionalString`

Reply:
Yeah I also found this today in my testing.